packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -459,12 +361,258 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
         });
     }
 
-    private async fetchSnapshot(snapshotOptions: string, tokenFetchOptions: TokenFetchOptions) {
+    private isTreesLatestSecondCallEvent(tokenFetchOptions: TokenFetchOptions) {
```

Comment:
"is" prefix means this function returns boolean, which is not the case.
Please rename to something that better reflects context

Reply:
Removed api.